src/Layout/GridWrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react';
+import styled, { DefaultTheme } from 'styled-components';
+
+interface GridWrapperProps {
+  /**
+   * Optional gap size to override the default and responsive gap sizes
+   */
+  gap?: keyof DefaultTheme['ws'];
+  /**
+   * The content to display inside the GridWrapper
+   */
+  children: ReactNode;
+}
+
+const StyledGridWrapper = styled.div<GridWrapperProps>`
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  gap: ${({ theme, gap }) => theme.ws[gap || 'small']};
+
+  @media (max-width: 768px) {
+    gap: ${({ theme, gap }) => theme.ws[gap || 'xsmall']}; /* Smaller gap for mobile devices */
```

Comment:
It looks like the `defaultProps` value below is breaking this conditional -- the value of `gap` will always be defined so it will never fall through to the `'xsmall'` option at smaller sizes. Maybe we can just change the default value to `null`?

Reply:
Makes sense, null seems like the better way to go.